packages/venly/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@web3-onboard/venly",
+  "version": "2.0.0-alpha.1",
+  "description": "Venly wallet module for connecting to Web3-Onboard. Web3-Onboard makes it simple to connect Ethereum hardware and software wallets to your dapp. Features standardised spec compliant web3 providers for all supported wallets, framework agnostic modern javascript UI with code splitting, CSS customization, multi-chain and multi-account support, reactive wallet state subscriptions and real-time transaction state change notifications.",
+  "keywords": [
+    "Ethereum",
+    "Web3",
+    "EVM",
+    "dapp",
+    "Multichain",
+    "Wallet",
+    "Transaction",
+    "Provider",
+    "Notifications",
+    "React",
+    "Venly",
+    "Connect Wallet",
+    "Ethereum Hooks",
+    "Blocknative",
+    "Crypto",
+    "Crypto Wallet"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/blocknative/web3-onboard.git",
+    "directory": "packages/venly"
+  },
+  "homepage": "https://onboard.blocknative.com",
+  "bugs": "https://github.com/blocknative/web3-onboard/issues",
+  "module": "dist/index.js",
+  "browser": "dist/index.js",
+  "main": "dist/index.js",
+  "type": "module",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc -w",
+    "type-check": "tsc --noEmit"
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "typescript": "^4.5.5"
+  },
+  "dependencies": {
+    "@venly/web3-provider": "^3.0.0-develop.8",
```

Comment:
Should this be pointed to a develop version? 
If there are any env build/bundler configs for  the @venly package that would be helpful to document in our core docs here: https://github.com/blocknative/web3-onboard/blob/develop/docs/src/routes/docs/%5B...3%5Dmodules/core.md#build-environments
For official release out of an `alpha` package the package will need to be able to be run within all of our example projects: https://github.com/blocknative/web3-onboard/blob/develop/docs/src/routes/docs/%5B...3%5Dmodules/core.md#build-environments
Testing for this will need to be done after `alpha` package publishing to get real-world instantiation/integration testing.

Reply:
Updated to latest stable build